### PR TITLE
Remove Helmet / Update addAlerts

### PR DESCRIPTION
**Pull request by Aringoaway**
### Changes

1. remove Helmet
2. update error alert messages

I will delete the Helmet in each module in the package.json

### CODE REVIEW

File: src/components/Credentials.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export function Credentials({ ...props }) {
 			const usernamesToLS = saveUsernamesToLS(response.data.data, credentials_ids, cleanupTime);
 			setCredentials(usernamesToLS);
 		} catch (e) {
-			props.app.addAlert("warning", t("ASABCredentials|Something went wrong, failed to fetch assigned credentials"));
+			props.app.addAlert("warning", t("ASABCredentials|Something went wrong, failed to fetch assigned credentials", {error: e?.response?.data?.message}), 30);
```
Comment (Pe5h4) [inline src/components/Credentials.js:30]: @Aringoaway This and all other refactored alert updates are wrong here. It should be like that:

```
props.app.addAlert("warning", `${t("ASABCredentials|Something went wrong, failed to fetch assigned credentials")}. ${e?.response?.data?.message}`, 30);
```
```

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Comment by Pe5h4** [comment]
@Aringoaway How does the app look like without helmet? (the tab)

**Comment by Aringoaway** [comment]
https://user-images.githubusercontent.com/65866093/220924700-e977ec9d-40f2-4fbf-af5f-60847cc3e494.mp4

**Review by Pe5h4 — APPROVED** [review]

**Comment by Pe5h4** [comment]
dont forget to update all apps with this change

**Pull request merged**